GOAT-Compiler/Code Generation/BuildInFunctionImplementations.cs

```diff
@@ -0,0 +1,343 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using GOAT_Compiler.Exceptions;
+
+namespace GOAT_Compiler.Code_Generation
+{
+    /// <summary>
+    /// The class with the implentations of the built in functions
+    /// </summary>
+    public class BuildInFunctionImplementations
+    {
+        private CNCMachine _machine;
+        private TextWriter _stream;
+        public BuildInFunctionImplementations(CNCMachine newMachine, TextWriter stream)
+        {
+            CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
+            _machine = newMachine;
+            _stream = stream;
+        }
+
+        /// <summary>
+        /// Calls the specified built in function with the right params.
+        /// </summary>
+        /// <param name="functionName">The name of the built in function.</param>
+        /// <param name="actuelParams">The params for the built in function.</param>
+        /// <returns></returns>
+        public dynamic CallBuildInFunctions(string functionName, List<dynamic> actuelParams)
```

Comment:
actuelParams is writen veru got, *actualParams

Reply:
renamed